helm/chaos-mesh/README.md

```diff
@@ -65,6 +65,14 @@ The following tables list the configurable parameters of the Chaos Mesh chart an
 | `dashboard.env.LISTEN_PORT` | | `2333` |
 | `dashboard.env.DATABASE_DRIVER`| The db drive used for Chaos Dashboard, support db: sqlite3, mysql| `sqlite3` |
 | `dashboard.env.DATABASE_DATASOURCE`| The db dsn used for Chaos Dashboard | `/data/core.sqlite` |
+| `dashboard.ingress.enabled`                   | Enable the use of the ingress controller to access the dashboard                         | `false`             |
+| `dashboard.ingress.certManager`               | Add annotations for cert-manager                                                      | `false`             |
```

Comment:
Could you please update this description?

Reply:
How about changing this description to `Enable the cert-manager for ingress` ?

@yeya24

Reply:
This is good. It is better to be `Enable Cert-Manager for ingress` since `the` is redundant.

Reply:
Done！